tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3493,4 +3493,335 @@ function test_get_styles_for_block_with_content_width() {
 		$style_rules = $theme_json->get_styles_for_block( $metadata );
 		$this->assertSame( $expected, $root_rules . $style_rules );
 	}
+
+	/**
+	 * @dataProvider data_generate_spacing_scale_fixtures
+	 *
+	 * @ticket 56467
+	 *
+	 * Tests generating the spacing presets array based on the spacing scale provided.
+	 */
+	function test_set_spacing_sizes( $spacing_scale, $expected_output ) {
+		$theme_json = new WP_Theme_JSON(
+			array(
+				'version'  => 2,
+				'settings' => array(
+					'spacing' => array(
+						'spacingScale' => $spacing_scale,
+					),
+				),
+			)
+		);
+
+		$theme_json->set_spacing_sizes();
+		$this->assertSame( $expected_output, _wp_array_get( $theme_json->get_raw_data(), array( 'settings', 'spacing', 'spacingSizes', 'default' ) ) );
+	}
+
+	/**
+	 * Data provider for spacing scale tests.
+	 *
+	 * @return array
+	 */
+	function data_generate_spacing_scale_fixtures() {
+		return array(
+			'one_step_spacing_scale' => array(
+				'spacingScale'    => array(
```

Comment:
```suggestion
				'spacing_scale'   => array(
```
🔢 Applies to other instances in the PR.

Reply:
The method expects the key to be `spacingScale` as this is what comes back from the theme.json parsing

Reply:
`spacingScale` in this context is the name of the test method's argument rather than the method being tested, which should match `$spacing_scale` in the test method's signature.